Assets/Scripts/ClickDamage.cs

```diff
@@ -20,6 +23,7 @@ void Update()
     {
         if(Input.GetMouseButtonDown(0) && canSwing)
         {
+            AudioSource.PlayClipAtPoint(punchHitSFX, Camera.main.transform.position);
```

Comment:
if this is only supposed to play if we hit an enemy, we should move it to line ~47. Probably a good idea so you know if you hit or not